### DISCUSSION TITLE
Aolithium Akku kompatibel?

### DISCUSSION THREAD

**Discussion by frankydee1980** [hoylabs/OpenDTU-OnBattery#1265] [Q&A]
Hallo,

es gibt von der Firma Aolithium zur Zeit preisgünstig 48V 100h Akkus.  Hat die schon jemand im Einsatz mit OpenDtu? Weiß jemand, ob er ausgelesen und verbunden werden kann?  (https://de.aolithium.com/)

Danke und Grüße

**Comment by spcqike** [comment]
![grafik](https://github.com/user-attachments/assets/eb96ee75-cd61-4d91-a713-794eb6e5a462)


da nehm ich gleich 5 :) 

sry, aber die Seite riecht für mich stark nach Scam. Auch wenn es keiner sein mag. aber warum sollte ich 999€ für einen bis dato no-name Akku ausgeben, wo ich nicht weiß, ob er kompatibel ist, wenn man bekannte Marken günstiger bekommt?
![grafik](https://github.com/user-attachments/assets/96c59887-42eb-4e04-a7b7-ea49d312e46b)


![grafik](https://github.com/user-attachments/assets/98b34ad5-8ff9-4b41-a747-9d961bba7f21)

Pseudorabatte hin oder her

**Reply by Manos1966** [reply]
@spcqike fuer mich bitte auch! 😜 

Leider ist die "Spar Berechnung" etwas ungluecklich... es ist nur der Kabelsatz 🙈 
![Capture](https://github.com/user-attachments/assets/1d8c5429-b2c9-47d7-ac8c-a137b258bcb0)

**Comment by PhilJaro** [comment]
Auf Ebay nochmals wesentlich günstiger...
![image](https://github.com/user-attachments/assets/2e001032-37f7-42e3-a289-6f348cc2da6c)

[Hier](https://youtu.be/9hZCcHFIdaI?si=rvuvFOqW05-epu_2) wurde das Teil mal geöffnet.

**Reply by spcqike** [reply]
ok, 200€ Unterschied zur Pylontech, da kann ich die Frage dann verstehen. Auch wenn die Webseite wirklich merkwürdig ist. in ihrer Art und Weise, viele nicht funktionierende Links und ein fehlendes Impressum.

**Comment by Manos1966** [comment]
@frankydee1980 Such die Produkt Daten, wenn die Batterie das Pylontech Protokoll unterstutzt, dann ist es zu 99,9% Prozent sicher, dass sie von der OpenDTU-OnBattery unterstützt wird. Und siehe da: "【Anpassung】: Kompatibel mit Pylontech, GoodWe, Deye, Megarevo, Victron Wechselrichtern." Etwas ungluecklich uebersetzt, gehe ich davon aus.

@spcqike Wo finde ich die Pylontech fuer 966EUR (link oder Shop Namen bitte).

@PhilJaro Danke fuer den Link. Fuer 755EUR ist es wirklich ein sehr guter Preis.

Schoen zu sehen, dass die Batterie Preise (sehr) schnell nach unten gehen. 💯 

Wenn wir von "Schnäppchen" reden, wirft Mal auch hier ein Blick (ich mag das eingebaute Loesch-System):
https://www.youtube.com/watch?v=9J58Vm6Ev3w

Am Ende des Tages, die Frage(n) sind: 
- Was passiert falls, fünf Jahre später, ein Garantie-Fall passiert
- Wie sicher fuehle ich mich, mit einer 20kWh "Bombe" eheem "Batterie" meinte ich 😆 , in meinem Haus 
(Ich gehe davon aus, wir sammeln alle Erfahrungen mit 3-5kWh z.Zeit. Die richtige Haus Installation benoetigt mindenstens 15 - 25kWh (Haus mit Waermepumpe oder eAuto).

**Reply by spcqike** [reply]
Die gibt es bei nkon

**Reply by tobi0171** [reply]
@Manos1966 zum Thema "Löschssystem"... hatte ich hier etwas unterhaltsammes dazu gesehen. https://www.youtube.com/watch?v=q1Poj1qzPmk

**Reply by Manos1966** [reply]
Es hat besser funktioniert als ich erwartet hätte.
Lieber im Haus mit Gestank zu leben, als in einem verbrannten Haus...

**Comment by Manos1966** [comment]
> Die gibt es bei nkon

NL ? Dann kostet der Versand (Selbstabholung nicht moeglich) 125EUR 🤦‍♂️

**Reply by spcqike** [reply]
Naja. Was kostet der Versand eines 40kg schweren Gerätes denn sonst so? Hab da keine Preise im Kopf, aber das Ding kommt doch sicher auf Palette mit Spedition, wie auch solar Paneele oder?

**Reply by Manos1966** [reply]
Deswegen fahre ich selber und hole sie 😉 
Aber bei NK NL geht es nicht (aufgrund der MWSt Befreiung) 😞

**Comment by frankydee1980** [comment]
Kleines Update. Ich konnte nicht widerstehen. Habe den Akku mit Rabatten für 724€ bestellt und er ist eben per Spedition angekommen.
Werde weiter berichten, ob ich ihn ausgelesen bekomme.

**Reply by PhilJaro** [reply]
Mit dem Code "Klarnaherbst" gehen sogar 704€ wenn der Code noch nicht verbraucht wurde. 
Bin auch schon kurz davor zuzuschlagen :D

**Comment by frankydee1980** [comment]
Also die Kommunikation funktioniert. Mit Can Bus Modul und Dipschalter 6 on.

Läuft!

**Reply by Manos1966** [reply]
@frankydee1980 wo hast du sie gekauft?

**Comment by frankydee1980** [comment]
@Manos1966  habe ich bei Ebay gekauft. https://www.ebay.de/itm/296620648071

**Reply by Manos1966** [reply]
Na Super... jetzt werde ich heute Nacht nicht schlafen koennen.... 😆😆😆

**Reply by tobi0171** [reply]
@frankydee1980 super das es auch mit dem Speicher klappt. 
Kannst du bitte einmal deine Einstellungen und den Anschluss beschreiben, wie es letzen Endes funktioniert hatte. 
Wie ist denn dein erstes Fazit nach 2 Monaten zu dem 5Kwh-Klötzchen... ich liebäugel tatsächlich auch grade als Nebenprojekt" damit, Preislich nochmal gefallen und echt verlockend. 
Hast du zufällig mal rein geschaut? Falls ja, welches BMS ist verbaut? Sind die Kabel mit dabei? Zerobrain hat dazu ja auch ein Video auf YT.  
Beste Grüße und Danke

**Reply by frankydee1980** [reply]
Hallo @tobi0171,
Anschlussplan mache ich dir nächste Woche und auch die Einstellungen.
Hab gestern ne Mail bekommen, dass der Speicher gerade 649€ kostet und mit Kabeln ist. Bin schon wieder in Versuchung, da ich bisher keine Probleme mit ihm habe.
Kein Auffälligkeiten, tut was er soll. 

Wenn du bestellst, bitte schauen das die Kabelsätze dabei sind(gerade ist Aktion, wenn du kaufst ist alles dabei). Die sind dummerweise anders wie bei Polyntech und nicht kompatibel. Reingeschaut habe ich nicht, weil da Siegel dran sind, zwecks Garantie.

Ein Ziel ist es noch den Speicher per RS485 auszulesen, um mehr Infos zu bekommen. Modul habe ich schon da, leider keine Zeit bisher.

Gruß Frank

**Reply by tobi0171** [reply]
@frankydee1980 Super danke dir.
Ja (08.11.2024) 599€ angeblich mit Kabeln bei ebay vom Verkäufer "aolithium usa" mit angeblicher Lieferurng aus DE. Anworten tun sie leider nicht ... aber ich glaub ich waage das Experiemnt dann auch mal. Bei deinem Link oben ist es scheinbatr nur der Speicher ohne Kabel.

**Reply by tobi0171** [reply]
@frankydee1980 

![image](https://github.com/user-attachments/assets/03f534e7-c88c-4ed8-9abe-13cd6afc2380)

Hab noch diese Info vom Hersteller bekommen. Vielleicht hilft das etwas weiter, falls noch nicht bekannt.

Sehe ich es richtig, du hast am Speicher (am Mäuseklavier) den PIN 6 auf "on" gestellt, den Rest auf "Off" gelassen, dann ein abgeschnittenes Netzwerkkabel an der einen Seite am CAN Anschluss verbunden und auf der anderen Seite, dort den PIN 4 und PIN 5, hiervon an der oDTUoB auf 4=H und 5=L angeschlossen?

Falls das so passt, hab ich's verstanden und bräuchte keine weitere Anleitung. 
(Das sollte dann aus der Beschreibung aus dem Bild ziemlich das Pylontech-Anschlussschema sein, korrekt?)

**Reply by frankydee1980** [reply]
Richtig. Dip 6 ist auf on, Rest off.

Ja, die Kommunikation ist etwas dürftig. Aber dieses pdf habe ich auch bekommen.

**Comment by tobi0171** [comment]
@frankydee1980 
Danke für den Tipp, die Erfahrung und den Test an der unbekannten Hardware.

Ich hab den Speicher heute auch erhalten (599€ inkl. Kabel und Gestellfüsse) per Spedition in einer Holzkiste. Erster Eindruck ist tatsächlich gut... schauen wir mal was die Zeit so bringt. 
Aktuell hängt er am Futternapf und ich hab mal mein Ersatz-Fusion mit ISO/CAN Adapter, was ich vorher so 1:1 am Pylontech so angeschlossen hatte, mit dem beiliegenden Kabel einfach mal angesteckt. Was soll ich sagen... scheint zu funktionieren 👌🏻... Dip 6 auf on, der Rest auf off... BMS über den Schalter neu gestartet und los ging's. 

![image](https://github.com/user-attachments/assets/fd98a687-e84a-4442-994a-18e2101dc0c9)![image](https://github.com/user-attachments/assets/d19dad79-e6c9-4b2b-a2b5-9041083f2de3)
![image](https://github.com/user-attachments/assets/566674c4-ef63-4be5-8b60-b00511583638)

Natürlich könnten ein paar Sachen wie "Blase unterm Display" und "besser abisolierte Kabel-Verbindungen" noch sein, aber ich denke bei dem Preis kann man nichts sagen.

**Reply by frankydee1980** [reply]
Sehr gut! Preis ist echt unschlagbar. Wenn ich das mit dem RS232 auslesen noch hinbekomme, sage ich dir Bescheid.

Gruß Frank

**Reply by Manos1966** [reply]
>  (599€ inkl. Kabel und Gestellfüsse)

Hast du den Deckel aufgemacht? Sind wirklich Batterie-Zellen drin? 🤣🤣🤣🤣🤣
TOP-Preis!!!! 🥇 👍

**Reply by tobi0171** [reply]
@Manos1966 Nein, das Klebe-Siegel und die markierten Schrauben hindern mich tastächlich noch etwas dran... aber vom Gewicht her sollten schon paar Zellen drin sein...( oder halt gut ausbetoniert 🤣🤣) > Aber der "Zerobrain" hat ja auf YouTube da vor 2 Monaten schon mal "reingeschaut" und das Teil getestet... daher auch das Interesse an dem Speicher, der Beitrag hier hat mich dann noch etwas mehr beflügelt und bestärkt.

**Reply by Manos1966** [reply]
>  oder halt gut ausbetoniert 

musstest du wirklich mich an der Zeit der HiFi Anlagen erinnern? 🙈🙈🙈🙈
...war gerade am berechnen wie viele 9V Batterien für 51,5V gebraucht werden 🤣 

Spass bei Seite: DANKE, dass du den Kauf-Test für uns gemacht hast!!!! ❤️

**Reply by tobi0171** [reply]
@Manos1966 Kein Problem, von Erfahrungen und Austausch lebt so ein Projekt ja. 
Der DANK geht eigentlich an @frankydee1980 als erstes "Versuchkanninchen" der ihn blind bestellt hat, ich wusste ja dadurch und YT schon ungefähr was auf mich zu kommt und das die Erfolgsaussichten ganz gut sind. 
Wenn jetzt noch das "PYLON" im Kopf verschwindet, und dort "Speicher" oder irgendwas anderes steht, kann mein innerer Monk auch noch gut schlafen ;)...

**Reply by tobi0171** [reply]
![s-l1600](https://github.com/user-attachments/assets/60800694-ad33-4c36-bd99-ece513e85437)

Kleiner Nachtrag.... Speicher ist nun voll und die Spannung liegt nach dem Abschalten bei 53,5V, die Zellen liegen laut Anzeige sogar recht nahe beieinander (zwischen 3342 und 3349 mV Zellspannung). Er läst auch keine weitere Ladung mehr zu..... 
Ich Esel hätte mal im HomeAssistant den SOC und die Spannung mitschreiben lassen sollen, während dem laden...aber da war die Euphorie wohl schneller.

**Reply by tobi0171** [reply]
Nachtrag 2... 
Nach einer Nacht "Entspannung" und ausbalancieren lassen, hab ich mich heute nochmal ans Nachladen mittels Labornetzteil (mit 56,5V 0,6A (weniger erkennt das BMS scheinbar nicht)) gemacht. Hierbei hab ich gleich mal die VenusOS-Kompatibilität mit gestestet, eigenlich unter der Premisse mir die Daten aufzeichnen zu können.  Wie man sieht... driften die Zellen doch im oberen Bereich aktuell noch sehr weit auseinander und das BMS macht scheibar erst bei 3,7V oder 3,75V (mir persönlich zu hoch) einen Cut.
 
Etwas "off Topic", aber falls jmd. ein paar Info`s hat, welches BMS verbaut ist und wie ich mich dort einschleichen kann, um ein paar Werte zu ändern, wäre ich dankbar. 

(Vollständigkeitshalber > Ich hab die Verkabelung + DIP Stellung 1:1 von der oDTUoB (mit Fusion-CAN-ISO) an derVenus verwendet (dort mit USB-CAN-Candellight-Adpater), hier kamen Daten, laut Handbuch sollten beim Victron jedoch alle DiPs auf OFF sein, was ich dann auch getan habe, hier ergeben sich dann noch ein paar mehr Daten. Leider aber auch nicht die einzelenen Zellspannungen. Der Name oben in Kopf wurde händisch vergeben, er hatte ihn als "Pylontech" erkannt/benannt.     

Nur zuf Info für die, die ggf damit Liebäugeln. 

![grafik](https://github.com/user-attachments/assets/caaadab8-5988-4067-959b-38bfced86da5)

**Reply by gitisgreat2023** [reply]
@tobi0171 
56.5V ist hoch, aber 308mV inbalance is wirklich viel. Insbesondere weil eine 3.693V hat... das wäre mir zu riskant dass da die Zelle permanent Schaden nimmt.
Ich vermute der balancer geht los bei 3.45V oder 3.50V. Hat er einen RS232 Schnittstelle? Mit dem ESP8266 board [lässt sich sehr schnell mittels WiFi und MQTT](https://github.com/hoylabs/OpenDTU-OnBattery/discussions/1391) dann die individuelle Zellspannung mitschreiben. Da im RS232 kann man auch sehen ob der balancer aktiv ist. Wenn nicht, auch okay einfach die Zellen über Zeit anschauen.
Oder halt über Victron es im Auge halten während Du langsam die Ladespannung hochschraubst.
Ich würde den mal 1-2 Tage bei 16*3.45=55.2V laufen lassen und den inbalance über Zeit anschauen. Wenn der schnell runter geht hat er einen aktiven balancer, wäre das beste. Pylontech hat leider einen passiven.

**Reply by tobi0171** [reply]
@gitisgreat2023 
Das wäre eine Super Idee, die Hardware hab ich sogar hier noch irgendwo vergraben vom Pylontech ... leider hat er nur CAN/RS485-500Kb Inverter, und RS485-B 9600bps Inverter, und RS485 Inverter, wobei die letzen beiden wohl eher für die Master/Slave-Geschichte sind wenn, sie im Stapel laufen.

Zur größten Not ... Siegelbruch und 5A Aktiv Balancer. Um das balancieren mach ich mir weniger Gedanken... eher stört mich, das er die Zellen teilweise bis 3,7-3,75V hochgehen lässt und erst dann abschaltet.... das würde ich gerne ändern.

**Reply by Manos1966** [reply]
Laut VICTRON Anleitung fuer Pylontech (15S waehrend deine Batterie 16S ist):

When DVCC is enabled, the battery (via the CAN-bms) is responsible for the charge voltage. 
The Pylontech battery requests a charge voltage of 53.2V. We have however found that in practice this is too high.

The Pylontech battery has 15 cells in series, so 53.2V **equates to 3.55V per cell**. **This is very highly charged** **_and makes the system prone to go overvoltage_**.

**It should also be noted that a LiFePO4 cell stores very little additional energy above 3.45V.**

For this reason we opted to override the BMS and cap the voltage at 52.4V. This sacrifices almost none of the capacity and greatly improves the stability of the system.

========================================================

Also lieber 3,45V x 16 = 55,2V (wie @gitisgreat2023 erklaert hat)


Die Spannung der einzelnen Zellen beobachten:
Ich benutze das Fork von Andreas @skippermeister Ich habe eine extra ESP32 OpenDTUonBattery mit RS485
https://github.com/hoylabs/OpenDTU-OnBattery/discussions/622#discussioncomment-8726163
Ich kann dir aber nicht sagen, ob es mit der Aolithium funktionieren wird.

**Reply by tobi0171** [reply]
@Manos1966 danke,
auch ein guter Vorschlag. Ich find den Fork von Skippermeister echt (schon seit Beginn) mega gut, bin aber nicht durch die Schaltpläne durchgestiegen, um es nachzubauen oder mir das Fusion entsprechende umzukrempeln. (Schaltpläne und tiefere Bauteilbeachaffenheiten sind nicht meins - Brachenfremd 😆)

((((Ich hab mich die Tage vorher mal mit AliExpress beschäftigt und hab das BMS von "QUCC" (?) gefunden, was optisch anhand der Anordnung usw halbwegs passend sein könnte und die mal angeschrieben. 
Dort gibt es auch einen passenden RS485-zu USB Adapter/Modul (bestellt) und entsprechende Software (bislang nicht gefunden)...aber die sind recht kommunikativ.. mal schauen ob es passt und ich da was bekomme, um die Zell-Abschaltspannung und die Zelldifferenz runter zu setzen. Vielleicht lasst sich ja ein Adapterkabel bauen, was RJ45 (485) auf das USB-Modul und die Software zulässt. 

Ob es das dann tatsächlich ist, weiß ich nicht, optisch kommt es aber halbwegs hin.

Dort, zumindest beim Typ 1 mit Display, sind auch Abschaltspannungen von 3,75V mit angegeben, was der Sache näher kommen könnte.))))

NACHTRAG: Scheint wohl ein Ke-Sheng? zu sein.

**Reply by tobi0171** [reply]
@frankydee1980 

Schau mal hier...RS232 wird nix, aber mitRS485 klappts ;)  . https://diysolarforum.com/threads/bemory-bms-ks48100.87902/?fbclid=IwZXh0bgNhZW0CMTEAAR3oKryP6IU6LTyyLZUv4DlJr3oyWQButb4I15qMgtKdH_FbEfG_d-PLGBA_aem_kUPuN2DXyj8ws6H5eX31MA

**Reply by ck873** [reply]
@frankydee1980
Danke für deine Fotos, die haben mir schonmal sehr weiter geholfen  :-)
Kannst du noch die Pin Belegung für den Anschluss des RJ45 Kabels (CAN) der AOLithium Batterie an das OpenDTU Fusion Board posten?

**Reply by frankydee1980** [reply]
@ck873 

für das Fusion Board brauchst du den CAN/ISO Shield. Dort dann an CanH den Pin 4 vom Cat Kabel anschließen und an CanL den Pin 5. Das sind die beiden mittleren Pins vom Cat Kabel. Und am Akku noch den Dipschalter 6 auf On

**Reply by tobi0171** [reply]
@ck873 hier noch mal ein Bild... Ist zwar für den Pylontech gewesen, funktioniert aber mit dem AOli genauso (siehe oben) 
Im Bild gelb CAN_L = 4 ; orange CAN_H = 5)...Dort mit RJ45-Buchse, so konnte ich dann einfach ein x-beliebiges Netzwerkkabel in passender länge nutzen. 
(Die anderen Kabel bitte erstmal nicht weiter beachten, dort sind noch zwei 4-Pin JST 2.0 PH Buchsenleitungen für die beiden MPPTs, die aus dem Gehäuse raus geführt sind angeschlossen.)
   
![grafik](https://github.com/user-attachments/assets/5035a003-99e8-425a-8110-52e371680e09)

**Comment by gitisgreat2023** [comment]
@tobi0171 
Schon mal den per bluetooth geschaut ob was da ist? Falls probiere mal einige Apps aus. Damit lässt sich vermutlich alles einstellen.
Es wurde mal [JDB](https://aolithium.com/blogs/aolithium-battery/what-bms-is-used-in-aolithium-batteries?srsltid=AfmBOoqXQv_SoNbD3RAqhqBzCSBNnbxG_jV50DsCAQ9_cY41a3t1M1wO) verbaut. Würde JKBMS und JDB mal probieren.

**Reply by tobi0171** [reply]
Moin @gitisgreat2023,ja hab ich schon getestet. BT leider nicht vorhanden. Was aber ggf. auch wieder für die Idee mit dem "QUCC" spricht, denn dort hat das Typ1 kein BT und das Typ2 verwendet den vorhandenen Anschluss wahlweise als LCD oder BT.

**Reply by tobi0171** [reply]
> Moin @gitisgreat2023,ja hab ich schon getestet. BT leider nicht vorhanden. Was aber ggf. auch wieder für die Idee mit dem "QUCC" spricht, denn dort hat das Typ1 kein BT und das Typ2 verwendet den vorhandenen Anschluss wahlweise als LCD oder BT.

Korrektur.... scheint ein "KS" zu sein, was auch immer das ist.

**Comment by Wooselman** [comment]
Hat schon jemand die AOLithium an einen Cerbo GX anschlossen mit dem Kabel, das dabei ist?
Funktioniert das oder braucht man ein spezielles Kabel?
Danke!

**Reply by tobi0171** [reply]
> Hat schon jemand die AOLithium an einen Cerbo GX anschlossen mit dem Kabel, das dabei ist?
> 
> Funktioniert das oder braucht man ein spezielles Kabel?
> 
> Danke!

An einen originalen Cerbo GX nicht, aber an ein RapiZero2WH+CAN-HAT und mit Venus OS 3.51. (Beitrag etwas weiter oben)
Ich hab das beiliegende Kabel verwenden. 
![image](https://github.com/user-attachments/assets/c3f75932-4e06-4eb2-b5f0-790600233a92)
"Trainiere" und monitore ihn grade mit Venus, da mir im oberen Bereich ü3,37V die Zellen auseinander driften.

**Reply by Wooselman** [reply]
Super, Danke. Den Beitrag oben hab ich gesehen, mir war nur nicht klar, ob es mit dem beiliegenden Kabel war 😉
dann noch viel Erfolg!

**Reply by tobi0171** [reply]
Nachtrag... für den MP2 hab ich das "VE.Can zu CAN-Bus BMS Typ A" Kabel in Verwendung.

**Comment by frankydee1980** [comment]
Falls jemand dachte der Preis war schon gut:

https://www.ebay.de/itm/226108733929?mkcid=1&mkevt=1&mkrid=707-53477-19255-0&campid=5338974865&toolid=10001&siteid=77&customid=ppr-de-2056012863

499€ mit Kabeln

**Comment by forumuser2018** [comment]
Hallo,

meine drei bestellten Einheiten sind angekommen und Sie wurden sofort korrekt erkannt.

![ao-1](https://github.com/user-attachments/assets/7b70539b-3026-4e6b-afd4-3d764d8aba0e)
![ao-2](https://github.com/user-attachments/assets/b90444f2-ef9f-4cc6-a261-21315dcbe1f8)

Verbindungskabel, Racks etc. waren alles dabei. Insgesamt liegen drei Satz Akku->Wechselrichter Stromkabel Sets dabei so das man jede Batterie auch Autark laufen lassen könnte oder die vollen 270A über eine Busbar nutzen könnte.

**Reply by tobi0171** [reply]
@forumuser2018 Sieht gut aus. Bin auf die Erfahrungen beim ausbalancieren gespannt... meine beiden waren sich da anfangs noch sehr sehr uneinig, aber scheinen sich nun nach knapp 2 Wochen, füttern und ruhen lassen, langsam einigen zu können.

**Reply by forumuser2018** [reply]
@tobi0171 Ich habe den Speicher spontan gekauft und ein Ladegrät ist bestellt und soll am Freitag kommen. Dann schliesse ich das ganze mal an und gucke wie sich die Akkus verhalten. Muss man tatsächlich erst einmal jeden Akku einzeln laden ? Aktuell haben die eine Ladung zwischen 20~25%. Ich wollte am Wochenende alle drei gleichzeitig laden. Das sollte doch das BMS dann ausgleichen oder nicht ?

**Reply by tobi0171** [reply]
@forumuser2018 ich musste sie 3-4 mal "zyklen" und langsam nach und nach laden bis sie einigemaßen beineinader gewesen sind....  bin noch immer dran.... aber langsam wird es. Bei deinen 15kWh wird das wohl etwas mehr Zeit in anspruch nehmen.
Anfangs waren es mehr als 350mV Differenz und rote Lämpchen bei 3756mV... aber nach ein paar schonenden Kuren und BMS-Eingriffen, scheints aber nun endlich zu klappen. 
Der erste (3 Wochen alt), ist inzwsichen okay, der "neue" (1 Woche alt) rebelliert noch, aber auch er wird es noch lernen.  
Schöner Nebeneffekt, bei der Lade- und Entladegeschichte unter Beobachtung.... die angegebene Kapazität kam ((gemessenen über einen Shunt) in etwa heraus. 

(Ich hab bei mir inzwischen den Cut-off runter gesetzt, den Balancer früher starten lassen und die Zellen-Differenz niedriger gesetzt .... da mir die Werkseinstellungen zu schroff gewesen sind, bin mir aber noch unschlüssig welche Einstellungen optimal sind)

**Reply by jo5177** [reply]
@tobi0171 Balancer früher starten lassen und Zellen-Differenz einstellen klingt aber so, als hättest du Zugriff auf´s BMS? Ist dem so?

**Reply by tobi0171** [reply]
> @tobi0171 Balancer früher starten lassen und Zellen-Differenz einstellen klingt aber so, als hättest du Zugriff auf´s BMS? Ist dem so?

Jawohl... lange gesucht und dann hier einen Weg gefunden. https://www.facebook.com/groups/866602481390913/

Man sollte aber halbwegs wissen, was man tut und verstellt.

**Comment by Wooselman** [comment]
Hallo,

wie sieht denn der Anschluss eurer Aolithium an den Inverter aus?
Ich möchte momentan nur eine betreiben. Brauche ich noch eine zusätzliche Fuse dazwischen?
Habe auch schon Setups gesehen, da wurde direkt an einen Victron MP2 angeschlossen.

Danke & Gruß!

**Comment by ck873** [comment]
Kann es sein, das die SOC Anzeige sich erst selbst kalibrieren muss? Ich hatte den Akku bisher erst einmal voll geladen und dann immer so ziwschen 50% und 20% gehalten. Nach einem Monat ist die SOC Anzeige denn dermaßen weggedriftet, dass ich bei 35% 49V hatte. Beim anschließenden Vollladen hat der Akku die 99% bereits nach 2kWh erreicht und 3,5kWh weiter geladen bis die 100% und 57V erreicht wurden. War oder ist das bei euch auch so?

**Reply by Detritalgeo** [reply]
Hallo, dies ist eine übersetzte Nachricht, also entschuldigen Sie, wenn die Worte nicht korrekt sind.

Das BMS kann nur mehr als 0,5 Ampere erkennen. Das bedeutet, dass kleine Lasten und Stromverbrauch im Leerlauf von der SOC-Schätzung nicht genau erfasst werden.

**Reply by gitisgreat2023** [reply]
Ich habe gerade meinen Pylontech verkauft. Aber auch der hatte SoC drift. Bei 91% SoC hat er immer kalibriert. Da hing er dann sehr lange zB also wirkliche SoC war niedriger als er getrackt hatte.

Wenn man nicht regelmäßig, zB ein mal pro Woche auf einer bestimmte Ladespannung mit kleinen Strom landet (i.e. 100% bei zB 3.45V/cell und nur 2A) sollte man lieber Spannungsgrenzen fürs Laden und Entladen nutzen. Auch dass nur wenn die Zellen einen Balancer haben und der Balance Drift nicht zu groß ist (oder ein gescheites BMS dass abschaltet).

Edit: wie oben beschrieben, Drifts kommen daher weil die Auflösung vom Strom nicht genau genug ist, kleine Ströme Werd zB gar nicht gezählt.

**Reply by tobi0171** [reply]
@ck873 Ich finde 2-3 komplette Ladezyklen nicht verkehrt, um dem BMS mal die Grenzen aufzuzeigen (natürlich nicht mit Vollgas),das hab ich bislang zumidest so gehandhabt. Ich hab jeweils 2-3 Zyklen von voll bis (fast) leer gemacht und auch den Balancer niedriger gestellt, das er früher tätig werden kann und dann langsam mit 10A bis zum Ende hin mit 1A geladen.  Bie mir hatte das BMS bei 6-9% Restkapazität einen Cut gemacht.  

@Detritalgeo ... Hi Mat.... Nice to see you here too... you can also be found everywhere where, who this battery is involved ;) ... thanks for the link to the DIY.Forum .to the KS_beta.

@gitisgreat2023 Kann ich bestätigen. Meine Pylontech hängen jetzt im Winter auch gerne bei 91% für längere Zeit fest. Daher habe ich die Entladegrenze entsprechend meiner Modulleistung stark nach oben (bei mir 85%) angepasst, das er gelegntlich mal die Chance bekommt überhaupt auf 100% zu kommen. Im Sommer nutze ich ihn von 99%/100% bis 25-30% Entladetiefe.

**Reply by ck873** [reply]
Danke für eurer Feedback. So ganz verstehe ich nicht warum das BMS sich an den Amper orientiert und nicht an der Spannung. Die Spannung muss zwar je nach dem was für ein Lade- oder Entladestrom fließt korrigiert werden, das scheint mir aber genauer als nur Lade- und Entladestrom größer 0,5A zu summieren. Wie dem auch sein, ich werde dann die SOC Werte ignorieren und nur nach Spannung regeln.

**Reply by Manos1966** [reply]
> Kann es sein, das die SOC Anzeige sich erst selbst kalibrieren muss?

Yes! depending on the Battery brand, it loses the calibration sooner or later.
One full charge will solve the problem.


In order to cope with the problem of SOC% losing precision, especially during the winter months, there is an Option in the DPL to disable SOC% information and use voltage data instead (much more precise):

![image](https://github.com/user-attachments/assets/77d67e7a-5f13-4460-981b-607ab1a55be6)

I do not remember if your Battery is a 15S or 16S 
Below is an estimate of the charge state % of your battery, based on voltage:
![image](https://github.com/user-attachments/assets/b27a6d4b-e9cd-495f-8a5e-6142d2730a75)

**Comment by frankydee1980** [comment]
Falls jemand seinen Akku loswerden will, ich würde einen nehmen zur Erweiterung. ;-)
